tests/unit/amo/reducers/test_users.js

```diff
@@ -312,6 +314,27 @@ describe(__filename, () => {
 
       expect(hasPermission(state, THEMES_REVIEW)).toEqual(true);
     });
+
+    it('returns `true` when user has a broad permission', () => {
+      const permissions = ['Addons:*'];
+      const { state } = dispatchSignInActions({ userProps: { permissions } });
+
+      expect(hasPermission(state, ADDONS_EDIT)).toEqual(true);
```

Comment:
These tests are difficult to verify during review because it's not obvious what `ADDONS_EDIT` actually is. I.e., it's not obvious that `ADDONS_EDIT` takes the form of `Addons:xxx`. Of course I could look it up, but it seems better to have a test that can easily be verified by just looking at the set up code.

It seems unnecessary to use an actual permission value in this test (in this case `ADDONS_EDIT`). You could just pass a string into `hasPermission`. If this line was changed to:

`expect(hasPermission(state, 'Addons:some_permission')).toEqual(true);`

then it would be more easy to verify that the test does what it claims to do.

The same holds true for the other two tests.